AjobK/Athena
# 374 Changed images of login and register

closes #363

<br />

 **What is it supposed to do**  
The pictures on the login and register screen needed to change. This PR adds two new pictures on the login and register screen.

<br />

 **How can we test your branch**  
1. Visit login screen, register screen
2. Notice changes

<br />

 **Additional notes**  
Things to keep in mind regarding the PR that don't fit in the previous two headings. 

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select at least two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

**Reviewer info**  
*1. You can use aliases to refer to specific PR checklist items. For example 'Hey @JohnDoe, for this line please check 3.a and 1.d.'*  
*2. Where possible, please use the suggestions feature in github so the author has a better idea of the solution you had in mind.*   

<br />

> This form should be filled in by the author of this PR.
